src/core/execute.c

```diff
@@ -4909,6 +4910,17 @@ static int exec_child(
                         return r;
         }
 
+        if (context->memory_ksm) {
+                if (prctl(PR_SET_MEMORY_MERGE, 1) < 0) {
+                        if (ERRNO_IS_NOT_SUPPORTED(errno))
+                                log_unit_debug_errno(unit, r, "KSM support not available, ignoring.");
+                        else {
+                                *exit_status = EXIT_KSM;
+                                return log_unit_error_errno(unit, errno, "Failed to enable KSM %m");
```

Comment:
enable → set (given you can also disable the feature with this, no?)

Reply:
Done.